### PR TITLE
Allow comma-separated list for SUPERNOVA_GROUP

### PR DESCRIPTION
**Pull request by msabramo**
E.g.:

```ini
[dfw]
SUPERNOVA_GROUP=raxusa,rackspace,dallas,usa
```

to make the `dfw` environment a member of all the listed groups.

Cc: @major, @sudarkoff

### CODE REVIEW

File: tests/configs/rax_without_keyring [hunk 1]
```diff
@@ -35,7 +35,7 @@ OS_USERNAME=my_username
 OS_TENANT_NAME=my_tenant
 
 [syd]
-OS_AUTH_SYSTEM=rackspace
+OS_AUTH_SYSTEM=rackspace,sydney,australia
```
Comment (major) [inline tests/configs/rax_without_keyring]: Hmm, did you mean to edit `OS_AUTH_SYSTEM` here?

File: tests/configs/rax_without_keyring [hunk 2]
```diff
@@ -46,7 +46,7 @@ OS_USERNAME=my_username
 OS_TENANT_NAME=my_tenant
 
 [hkg]
-OS_AUTH_SYSTEM=rackspace
+OS_AUTH_SYSTEM=rackspace,hong_kong,china
```
Comment (major) [inline tests/configs/rax_without_keyring]: Same question as line 39.

**Comment by major** [comment]
Couple of questions in-line.

File: tests/configs/rax_without_keyring (hunk 1 above)
Comment (msabramo) [inline tests/configs/rax_without_keyring]: Good catch. Fixing this...

File: tests/configs/rax_without_keyring (hunk 2 above)
Comment (msabramo) [inline tests/configs/rax_without_keyring]: Good catch. Fixing this...

**Comment by msabramo** [comment]
Fixed the 2 typos (thanks!) and rebased on the latest `master`.

**Pull request merged**